External/FEXCore/Source/Interface/Core/JIT/x86_64/ALUOps.cpp

```diff
@@ -671,6 +671,16 @@ DEF_OP(Extr) {
   }
 }
 
+DEF_OP(PExt) {
+  auto Op = IROp->C<IR::IROp_PExt>();
+
+  const auto Input = static_cast<Xbyak::Reg32e>(GRS(Op->Args(0).ID()));
```

Comment:
You likely want something like `const auto Input = OpSize == 4 ? GRS(Op->Args(0).ID()).cvt32() : GRS(Op->Args(0).ID())` But more clean